### PR TITLE
Responsive

### PR DESCRIPTION
**Pull request by ghost**
## Summary

- Responsive design

## Screenshots

Captura de pantalla de 2019-05-27 12-35-06

## Trello Card

https://trello.com/c/O3C99KiB/15-dise%C3%B1o-responsive

### CODE REVIEW

File: maquetado/scss/navbar.scss [hunk 1]
```diff
@@ -20,3 +20,13 @@
...
+
+@media only screen and (max-width: 1024px) {
+  .navbar {
+    position: relative;
+  }
+
+  .navbar-text{
+    display: none;
+  }
+}
```
Comment (mmamaniwolox) [inline maquetado/scss/navbar.scss]: tailing line

File: maquetado/scss/tech.scss [hunk 2]
```diff
@@ -58,3 +54,18 @@ $tech-logo-size: 100px;
...
+
+  .wrap {
+    display: inline;
+  }
+
+  .first-card {
+    border: none;
+    max-height: 350px;
+  }
+}
```
Comment (mmamaniwolox) [inline maquetado/scss/tech.scss]: tailine line

File: maquetado/scss/tech.scss [hunk 3]
```diff
@@ -58,3 +54,18 @@ $tech-logo-size: 100px;
...
+  .technology {
+    border-top: 1px solid $green;
+  }
+
+  .wrap {
+    display: inline;
+  }
+
+  .first-card {
+    border: none;
```
Comment (mmamaniwolox) [inline maquetado/scss/tech.scss]: just to learn, instead of using this, add a pseudo-element to technolgy, something like `nth-child` or `first-child`

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

File: maquetado/index.html [hunk 4]
```diff
@@ -15,33 +15,33 @@
     <h2 class="navbar-text">Developer Tools</h2>
   </div>
   <h2 class="title">Front-End Tools</h2>
-  <div class="technologies">
-    <div class="wrap">
+  <div class="container">
+    <div class="technologies">
       <div class="technology first-card">
         <img class="logo" src="./assets/react_icon.svg">
```
Comment (Dorian30) [inline maquetado/index.html]: Same as previos PRs. I would modify this into a more specific class like `tech-logo`

File: maquetado/scss/tech.scss [hunk 5]
```diff
@@ -54,3 +50,22 @@ $tech-logo-size: 100px;
...
+  .technologies {
+    display: inline;
+  }
+
+  .wrap {
+    display: inline;
+  }
+
+  .first-card {
+    border: none;
```
Comment (mmamaniwolox) [inline maquetado/scss/tech.scss]: just to learn, instead of using this, add a pseudo-element to technolgy, something like nth-child or first-child

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by Dorian30 — APPROVED** [review]

File: maquetado/scss/navbar.scss [hunk 6]
```diff
@@ -19,3 +19,13 @@
   font-size: $text-big;
   margin: 0 20px;
 }
+
+@media only screen and (max-width: 1024px) {
+  .navbar {
+    position: relative;
+  }
+
+  .navbar-text{
```
Comment (mmamaniwolox) [inline maquetado/scss/navbar.scss]: add space before `{`

File: maquetado/scss/tech.scss [hunk 7]
```diff
@@ -54,3 +54,18 @@ $tech-logo-size: 100px;
...
+  .technology {
+    border-top: 1px solid $green;
+  }
+
+  .technologies {
+    display: inline;
+  }
+
+  .technology:first-child {
+    border:none;
```
Comment (mmamaniwolox) [inline maquetado/scss/tech.scss]: add space after `:`

**Review by mmamaniwolox — CHANGES_REQUESTED** [review]

**Review by mmamaniwolox — APPROVED** [review]

**Pull request merged**